plugins/GSdx/GSRendererHW.cpp

```diff
@@ -64,14 +64,14 @@ void GSRendererHW::SetScaling() {
 	
 	m_upscale_multiplier = theApp.GetConfig("upscale_multiplier", m_upscale_multiplier);
 
-	if (m_upscale_multiplier == 1 && !m_nativeres) { //Custom
+	if (m_upscale_multiplier == 7) { //Custom Resolution
 		m_width = theApp.GetConfig("resx", m_width);
 		m_height = theApp.GetConfig("resy", m_height);
 	}
 		
-	if (m_upscale_multiplier > 1)
+	if (m_upscale_multiplier > 0 && m_upscale_multiplier !=7)  
 	{
-		if (m_upscale_multiplier > 6)
+		if (m_upscale_multiplier > 7)  // If users set more than 7 in INI file.
```

Comment:
hmm, that would allow frame buffer size to higher multiples. higher value customization would be possible this way, I think I will do that.